practice1/pr1.html

```diff
@@ -37,30 +37,88 @@
         </div>
         <hr>
     <section class="blocks-layout">
-        <div class="block-caption">
-            <h3 class="secondary-header">Spectacles designed for faces. The average ones, the unaverage ones and the very unaverage ones too.</h3>
-            <a href="">Shop spectacles</a>
+        <div>
+            <h3 class="secondary-header">Spectacles<br>designed for faces.<br>The average ones, the unaverage ones and the very unaverage ones too.</h3>
+                <a href="" class="caption-small shop-btn-container"><img src="img/Frame (2).svg" alt="Shop here" class="shop-btn" />Shop spectacles</a>
         </div>
         <div class="block-bradley img-text">
-            <h4 class="caption-small">Narrow bridge</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Narrow bridge</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 2.png" alt="Bradley" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Bradley</a>
+                <span>1 565 kr</span>
+            </div>
         </div>
         <div class="block-gifford img-text">
-            <h4 class="caption-small">Adjustable nose pads</h4>
+            <div class="caption-favorite">
+                <h4 class="caption-small">Adjustable nose pads</h4>
+                <img src="img/Frame (1).svg" alt="Favorite" width="16" height="16" />
+            </div>
             <img src="img/Rectangle 3.png" alt="Gifford" />
-            <span></span>
+            <div class="block-caption caption-small">
+                <a href="">Gifford</a>
+                <span>1 565 kr</span>
+            </div>
         </div>
         <div></div>
```

Comment:
It seems you're skipping a grid cell with this **div** 👮 
This is a violation of Geneva convention 🌐 
Instead, you can name you grid lines and assign elements to them via `grid-column` and `grid-row`

Reply:
Missed the line here 😔 
I'm talking about the empty `div` on line 66